Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:latest
+
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json /usr/src/app/
+RUN npm install -g micro
+RUN npm install
+
+# Copy app source code
+COPY . /usr/src/app
+
+#Expose port and start application
+CMD [ "npm", "start" ]
```

Comment:
my preferred convention is to have an `yarn start:prod` (npm run start:prod) for production and an `yarn start` for development.